README.md

```diff
@@ -133,6 +133,9 @@ deleting trailing white spaces before saving (via Emacs'
 (setq super-save-delete-trailing-whitespaces 'except-current-line)
 ```
 
+By default, `super-save` will automatically save only the current buffer, if you
```

Comment:
You might mention below some of your rationale, so people would get a better idea why this might be useful:

> I can confirm that having an option to save all buffers can be interesting, specially when doing indirect buffer edits (like when editing greps with occur-mode and occur-edit-mode, or when running a project-wide search and replace project-query-replace-regexp, etc.). In these cases, we can indirectly edit several buffers without actually visiting or switching to these buffers.